packages/babel-plugin-proposal-class-properties/test/fixtures/private/tagged-template-static/exec.js

```diff
@@ -0,0 +1,17 @@
+class Foo {
+    static #tag() {
+      return this;
+    }
+    
+    static #tag2 = () => this
+
+    static getReceiver() {
+        return this.#tag``;
+    }
+      
+    static getReceiver2() {
+        return this.#tag2``;
+    }
+}
+  
+expect(Foo.getReceiver()).toBe(Foo.getReceiver2());
```

Comment:
In this test we don't need also the private method, we can just compare the receiver with what we expect:
```suggestion
expect(Foo.getReceiver2()).toBe(Foo);
```

otherwise the test will pass even if both are broken (for example, if both return `undefined`).

Reply:
Yup. Let's break this into two `expect` calls, each asserting that the return is `Foo`.